iguana/m_notary_build

```diff
@@ -1,10 +1,4 @@
 #!/bin/bash
 pkill -15 iguana
-rm -f ../agents/iguana *.o
-cd secp256k1; ./m_unix; cd ..
-cd ../crypto777; ./m_LP; cd ../iguana
-#gcc -g -fno-aggressive-loop-optimizations -Wno-deprecated -c -O2 -DISNOTARYNODE=1 -DLIQUIDITY_PROVIDER=1 *.c ../basilisk/basilisk.c ../gecko/gecko.c ../datachain/datachain.c
-clang -g -Wno-deprecated -c -O2 -DISNOTARYNODE=1 *.c ../basilisk/basilisk.c ../gecko/gecko.c ../datachain/datachain.c
-#gcc -g -fno-aggressive-loop-optimizations -Wno-deprecated -c -DISNOTARYNODE=1 -DLIQUIDITY_PROVIDER=1  main.c iguana777.c iguana_bundles.c ../basilisk/basilisk.c
-clang -g  -Wno-deprecated -c -DISNOTARYNODE=1 main.c iguana777.c iguana_bundles.c ../basilisk/basilisk.c
-clang -g -o ../agents/iguana *.o ../agents/libcrypto777.a -lnanomsg -lcurl -lssl -lcrypto -lpthread -lz -lm -lsodium
+make clean
+make
```

Comment:
`m_notary_build` throws an error since the changes - https://github.com/KomodoPlatform/dPoW/pull/419#issuecomment-1494624469
Should we remove it completely? 
btw this one has been merged, dev -> master PR here https://github.com/KomodoPlatform/dPoW/pull/456

Reply:
m_notary_build does not compile any more: https://github.com/KomodoPlatform/dPoW/pull/419#issuecomment-1494624469

Reply:
@smk762 @cipig As I mentioned above (sorry, I edited post few times, seems that's why you didn't see full comment) to make old `m_notary_build` work again, just need to remove `cards777.c`.